src/components/navbar/Navbar.jsx

```diff
@@ -13,18 +13,16 @@ import {
   QUESTIONS_ROUTE,
 } from '../../routes';
 import AddQuestion from '../modals/AddQuestion/AddQuestion';
-import SignUp from '../modals/SignUp/SignUp';
-import SignIn from '../modals/SignIn/SignIn';
+// import UserAuth from '../userSection/UserAuth';
```

Comment:
Let's remove any unused importes.

Reply:
We will use this in a ternary operator to switch between the user name and pic section on the navbar if there is a user or keep the sign in and sign up section if there is no user